documentation/modules/selecting-migration-network-for-vmware-source-provider.adoc

```diff
@@ -3,8 +3,8 @@
 // * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
 
 :_content-type: PROCEDURE
```

Comment:
If this module only discusses VMware, change the module's name as well.

Reply:
But it is 'Selecting a migration network for a VMware source provider'.

Reply:
The module's name = the file's name, not the text.  The ID should be the same as the file name whenever possible.

Reply:
@RichardHoch 
 The file name is "selecting-migration-network-for-vmware-source-provider.adoc"
The ID is "selecting-migration-network-for-vmware-source-provider"
The title is "Selecting a migration network for a VMware source provider"
What is wrong? Or am I especially obtuse today?

Reply:
Sorry. My mistake. LGTM. Please make these changes on the PR for 2.2.